pkg/storage/unified/client.go

```diff
@@ -27,7 +32,9 @@ import (
 	"github.com/grafana/grafana/pkg/storage/unified/sql"
 )
 
-const resourceStoreAudience = "resourceStore"
+const (
+	resourceStoreAudience = "resourceStore"
+)
```

Comment:
nit: to minimize diff, this can be reverted.

Reply:
Done 👍